src/test/java/uk/gov/hmcts/reform/professionalapi/controller/advice/ExceptionMapperTest.java

```diff
@@ -35,29 +31,34 @@
 
     @Test
     public void should_handle_empty_result_exception() {
-        EmptyResultDataAccessException emptyResultDataAccessException = mock(EmptyResultDataAccessException.class);
+        EmptyResultDataAccessException emptyResultDataAccessException = new EmptyResultDataAccessException(1);
 
         ResponseEntity<Object> responseEntity = exceptionMapper.handleEmptyResultDataAccessException(emptyResultDataAccessException);
 
         assertEquals(HttpStatus.NOT_FOUND, responseEntity.getStatusCode());
+        assertEquals(emptyResultDataAccessException.getMessage(), ((ErrorResponse)responseEntity.getBody()).getErrorDescription());
     }
 
     @Test
     public void should_handle_resource_not_found_exception() {
-        ResourceNotFoundException resourceNotFoundException = mock(ResourceNotFoundException.class);
+        ResourceNotFoundException resourceNotFoundException = new ResourceNotFoundException("Resource not found");
```

Comment:
there is no test for ResourceNotFoundException

Reply:
This is the test for ResourceNotFoundException - I replaced the mock of the exception with a real one.

Reply:
i meant there is no unit test class for this

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/450/commits/7b95444d80f2c24770c02fdc7b09f5f5ea9aa765